recipes/glu/all/conanfile.py

```diff
@@ -0,0 +1,65 @@
+from conans import ConanFile, tools
+from conans.errors import ConanException
+import os
+
+class SysConfigGLUConan(ConanFile):
+    name = "glu"
+    version = "system"
+    description = "cross-platform virtual conan package for the GLU support"
+    topics = ("conan", "opengl", "glu")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.opengl.org/"
+    license = "unspecified"
```

Comment:
```suggestion
    license = "SGI-B-2.0"
```
https://cgit.freedesktop.org/mesa/glu/tree/include/GL/glu.h#n2